src/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -1,10 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks Condition="'$(TargetsWindows)' == 'true'">true</AllowUnsafeBlocks>
+    <DefineConstants Condition="'$(TargetsNetCoreApp)' != 'true'">$(DefineConstants);INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
```

Comment:
This shouldn't be needed anymore, you can just add a `<Nullable>annotations</Nullable>` property and you'll get this constant and the `NullableAttributes.cs` file imported when the configuration needs it:

https://github.com/dotnet/corefx/blob/cb5be3e9c76c337cb1752b206a50ffc71947aba1/src/Directory.Build.targets#L12-L21